.github/workflows/k6-perf-test-cloud.yaml

```diff
@@ -0,0 +1,35 @@
+name: K6 Performance Cloud Test
+on:
+  workflow_dispatch:
+
+jobs:
+
+  wallet-creation:
+    runs-on: ubuntu-latest
+    env:
+      CARDANO_SERVICES_URL: ${{ secrets.CARDANO_SERVICES_URL }}
```

Comment:
```suggestion
      PROVIDER_SERVER_URL: ${{ secrets.CARDANO_SERVICES_URL }}
```

Reply:
addressed in [edafbf1](https://github.com/input-output-hk/cardano-js-sdk/pull/710/commits/edafbf1b732a088391aad5fd223ef67542a6eeaf)